ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -571,7 +571,7 @@ const NetworksForm = ({
         />
         <FormField
           error={errors.rpcUrl?.msg || ''}
-          onChange={setRpcUrl}
+          onChange={(e) => setRpcUrl(e.replace(/\s+/gu, ''))}
```

Comment:
Arrow function definition should be avoided in props. Could you define it outside ?

Reply:
@GuillaumeRx defining it outside or in props has the same effect, from a re-rendering point of view. Could you explain more about what you are attempting to avoid by not having the arrow function in props?